website/siteConfig.js

```diff
@@ -77,7 +77,18 @@ const siteConfig = {
   },
 
   // Add custom scripts here that would be placed in <script> tags.
-  scripts: ['https://buttons.github.io/buttons.js'],
+  scripts: [
+    'https://buttons.github.io/buttons.js',
+    '/js/load.js',
+    {
+      src: '/js/config.js',
+      defer: true,
+    },
+    {
+      src: '/js/klaro.min.js',
```

Comment:
Good idea to use the minified version. I see this PR includes both the minified and original versions of the file though. Do we need both?

Reply:
I agree, the original version is not very readable either, we could just drop it